extra/1-currency-conversion.js

```diff
@@ -15,7 +18,11 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(currency) {
+  var addFee = currency * 0.01;
+  var foreignRate = (currency - addFee) * 5.7;
```

Comment:
Very minor point, but I think it would be better to declare 5.7 as the variable foreignRate, and then have a separate variable for the final result, as this makes the code easier to understand

Reply:
Thanks Karen. This practice will make the code more understandable. I've made the changes.